### PR TITLE
Release v1.11.0

### PR DESCRIPTION
**Pull request by theishshah**

### CODE REVIEW

File: changelog/generated/v1.11.0.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## v1.11.0
+
+### Additions
+
+- (go/v3, ansible/v1, helm/v1) Added `containerPort` protocol field required for server-side apply in manifests. ([#4863](https://github.com/operator-framework/operator-sdk/pull/4863))
+- Added a predicate for helm operators that filters resources based on selectors specified in watches.yaml. Only the resources that contain the labels specified by selectors will be reconciled. ([#4997](https://github.com/operator-framework/operator-sdk/pull/4997))
```
Comment (varshaprasad96) [inline changelog/generated/v1.11.0.md:6]:
```suggestion
- Fro helm based operators, added a predicate that filters resources based on selectors specified in `watches.yaml`. Only the resources that contain the labels specified by selectors will be reconciled. ([#4997](https://github.com/operator-framework/operator-sdk/pull/4997))
```

File: changelog/generated/v1.11.0.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- For helm-based operators, support go `text/template` expansion of override values. ([#5105](https://github.com/operator-framework/operator-sdk/pull/5105))
+
+### Changes
+
+- (go/v3) Fixed a bug in `make test` caused by incorrect envtest asset setup. ([#4863](https://github.com/operator-framework/operator-sdk/pull/4863))
+- (go/v3, ansible/v1, helm/v1) Increased resource limits for controller manager. ([#4863](https://github.com/operator-framework/operator-sdk/pull/4863))
+- Updated operator-framework dependency to 0.10.5. This specific update includes a fix for bundle validation when checking for invalid service accounts. ([#5119](https://github.com/operator-framework/operator-sdk/pull/5119))
+- For helm-based operators, reverted the following PRs which modified helm release equality comparison. - https://github.com/operator-framework/operator-sdk/pull/5042 - https://github.com/operator-framework/operator-sdk/pull/4937. ([#5097](https://github.com/operator-framework/operator-sdk/pull/5097))
```
Comment (varshaprasad96) [inline changelog/generated/v1.11.0.md:14]:
```suggestion
- For helm-based operators, reverted #5042 and #4937  which modified helm release equality comparison. ([#5097](https://github.com/operator-framework/operator-sdk/pull/5097))
```

**Review by varshaprasad96 — APPROVED** [review]
/lgtm

**Comment by laxmikantbpandhare** [comment]
testdata not generated.

**Review by laxmikantbpandhare — APPROVED** [review]

**Pull request merged**